src/assets/scss/_reset.scss

```diff
@@ -0,0 +1,95 @@
+@import url('https://fonts.googleapis.com/css2?family=Nunito+Sans:wght@300;600;800&display=swap');
```

Comment:
Dlaczego w ten sposób ładujesz fonty? Wiesz, co znaczy `display=swap`? Z reguły lepiej jest ładować fonty w inny sposób, polecam poczytać tutaj

https://web.dev/font-best-practices/#avoid-using-preload-to-load-fonts

Reply:
dodane do head